### PR TITLE
Remove Comma

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Remove comma if style is not set

See #9116

### CODE REVIEW

**Comment by UncleR** [comment]
I have tested this item :red_circle: unsuccessfully on 92ae51f0af53923a79c9173797ead05cc56dc162<br /><br />After chosing a module nothing happened. Returning to the editor was only possible with the close-Button of the selection window.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

File: administrator/components/com_modules/views/modules/tmpl/modal.php [hunk 1]
```diff
@@ -46,15 +46,22 @@
...
+		} else {
+			extraVal =  "";
 		}
 
-		modulePosIns = function(position) {
-			parent.window.jInsertEditorText("{loadposition " + position + "," + jQuery("#extra_class").val() + "}", "' . $editor . '");
-			parent.window.jModalClose();
-		}
+	        moduleIns = function(type, name) {
+	            parent.window.jInsertEditorText("{loadmodule " + type + "," + name + extraVal + "}", "' . $editor . '");
```
Comment (richard67) [inline administrator/components/com_modules/views/modules/tmpl/modal.php]: Shouldn't it be **$**extraVal here?

**Comment by richard67** [comment]
I have tested this item :red_circle: unsuccessfully on 92ae51f0af53923a79c9173797ead05cc56dc162<br /><br />Same as for @UncleR , it worked before this patch.

Following error is reported in browser console:

	TypeError: jQuery(...).val(...) is undefined
	
	if (jQuery("#extra_class").val().length)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Comment by richard67** [comment]
@dgt41 See [https://github.com/dgt41/joomla-cms/pull/29](https://github.com/dgt41/joomla-cms/pull/29 "https://github.com/dgt41/joomla-cms/pull/29") for the correction :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Comment by richard67** [comment]
@dgt41 Sorry, with my PR dgt41#29 mentioned above it also does not work. The Javascript error is solved, but the value of the extra class input field is never used with it. I try to find a solution but am neither a Javascript nor a jQuery expert.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Comment by richard67** [comment]
@dgt41 I have just commited a change to my PR dgt41#29 for you, and now all works, just have tested. So if you accept my PR this PR here can be tested with success.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Comment by dgt41** [comment]
@richard67 thanks, of the patch, I've simplified these lines a bit more

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 29f9e6b4b3fcce972d58e18b33f6531e0dee9924<br /><br />Tested with success.
Extra style parameter is added to the loadmodule or loadposition call at the end, separated by a comma, if set in the input box, otherwise if no extra style then no parameter and no comma added.
Examples:
{loadmodule mod_languages,Language switcher}
{loadmodule mod_languages,Language switcher,dada}
{loadposition position-0}
{loadposition position-0,dada}
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Comment by viocassel** [comment]
I have tested this. Everything works fine.

**Comment by UncleR** [comment]
I have tested this item :white_check_mark: successfully on 29f9e6b4b3fcce972d58e18b33f6531e0dee9924<br /><br />OK.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Comment by brianteeman** [comment]
Thanks for testing - setting RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9117">issues.joomla.org/joomla-cms/9117</a>.</sub>

**Pull request merged**